app/build.gradle

```diff
@@ -219,3 +200,25 @@ def gitSha() {
 def buildTime() {
     return new Date().format("yyyy-MM-dd'T'HH:mm'Z'", TimeZone.getTimeZone("UTC"))
 }
+
+def hasSigningConfig(String flavor) {
+    return project.hasProperty("signing.$flavor-release.keystoreFilePath") &&
+            project.hasProperty("signing.$flavor-release.keystorePassword") &&
+            project.hasProperty("signing.$flavor-release.keyAlias") &&
+            project.hasProperty("signing.$flavor-release.keyPassword")
+}
+
+def setSigningConfig(String flavor) {
+    if (hasSigningConfig(flavor)) {
+        def props = project.getProperties()
+        def signingConfig = android.signingConfigs[flavor]
+        signingConfig.storeFile file(props["signing.$flavor-release.keystoreFilePath"])
+        signingConfig.storePassword props["signing.$flavor-release.keystorePassword"]
+        signingConfig.keyAlias props["signing.$flavor-release.keyAlias"]
+        signingConfig.keyPassword props["signing.$flavor-release.keyPassword"]
+    }
+}
+
+setSigningConfig("ccc35c3")
+setSigningConfig("cccamp2019")
+setSigningConfig("ccc36c3")
```

Comment:
Can you please iterate all product flavors and automatically set the signing config based on their flavor name (similar to above comment). Pseudo code:

``` groovy
def setSigningConfigs() {
   for (flavor : flavors) {
     setSigningConfigs(flavor name)
   }
}

setSigningConfigs()